x-pack/plugin/otel-data/src/main/resources/component-templates/semconv-attribute-to-ecs@mappings.yaml

```diff
@@ -0,0 +1,32 @@
+---
+version: ${xpack.oteldata.template.version}
+_meta:
+  description: Aliases from OpenTelemetry SemConv attribute fields to ECS (and some non-ECS) fields
+  managed: true
+template:
+  mappings:
+    properties:
+      attributes:
+        type: passthrough
+        dynamic: true
+        priority: 10
+        time_series_dimension: true
+        properties:
+          exception.type:
+            type: keyword
+            ignore_above: 1024
+          exception.message:
+            type: keyword
+            ignore_above: 1024
+          exception.stacktrace:
+            type: keyword
+      error.exception.type:
+        type: alias
+        path: attributes.exception.type
+      error.exception.message:
+        type: alias
+        path: attributes.exception.message
+      error.exception.stacktrace:
+        type: alias
+        path: attributes.exception.stacktrace
```

Comment:
I'm not sure that alias is appropriate. The `error.exception.stacktrace` field is an (unmapped) object with a specific structure in the classic APM data streams. For OTel data streams, `attributes.exception.stacktrace` is a string containing the stack trace as a plain string.

See https://github.com/elastic/elasticsearch/blob/4ce91a943a6000ece125d51a922456219905d03b/x-pack/plugin/apm-data/src/main/resources/component-templates/logs-apm.error%40mappings.yaml#L19-L21

Reply:
Right.. 

I remember there was already some handling of Java OTel stack traces done. Was it parsing the string back to this format? Maybe my memory is wrong, but wasn't there something done for handling stack traces as string?

I opened a slack thread about it - maybe the UI can handle it. If UI devs say no, then this is indeed an issue - then probably best is to just remove this mapping - but then stacktrace will be likely just ignored, right?

Reply:
> Based on https://github.com/elastic/apm-data/blob/bd5e34116889942c53b186deab0f76beaf182de0/input/otlp/exceptions.go#L94 and

Oh, nice 👍

Reply:
`error.stack_trace` is in place now.